velox/functions/sparksql/fuzzer/tests/SparkQueryRunnerTest.cpp

```diff
@@ -80,6 +82,24 @@ TEST_F(SparkQueryRunnerTest, DISABLED_basic) {
   exec::test::assertEqualResults(sparkResults, outputType, {expected});
 }
 
+// This test requires a Spark coordinator running at localhost, so disable it
+// by default.
+TEST_F(SparkQueryRunnerTest, DISABLED_decimal) {
+  auto aggregatePool = rootPool_->addAggregateChild("decimal");
+  auto queryRunner = std::make_unique<fuzzer::SparkQueryRunner>(
+      aggregatePool.get(), "localhost:15002", "test", "decimal");
+  auto input = makeRowVector({
+      makeConstant<int128_t>(123456789, 25, DECIMAL(34, 2)),
+  });
+  auto outputType = ROW({"a"}, {DECIMAL(34, 2)});
+  auto sparkResults =
+      queryRunner->execute("SELECT abs(c0) FROM tmp", {input}, outputType);
+  auto expected = makeRowVector({
```

Comment:
nit: we don't need `expected`.  Can we use `input`?

Reply:
Thanks for your review. Updated.